views/default/admin/upgrades/list.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Lists all upgrades waiting to be processed
+ */
+
+elgg_require_js('elgg/admin/upgrade');
+
+$upgrades = elgg_extract('upgrades', $vars);
+
+$upgrade_list = '';
+foreach ($upgrades as $upgrade) {
+	$upgrade_list .= elgg_view('admin/upgrades/list_item', array('upgrade' => $upgrade));
+}
+
+$action_link = elgg_view('output/url', array(
+	'text' => elgg_echo('upgrade'),
+	'href' => 'action/admin/upgrade',
+	'class' => 'elgg-button elgg-button-action mtl',
```

Comment:
add the 'hidden' class here - so we don't give a clickable button until the js is ready to go